packages/import-maps-generate/README.md

```diff
@@ -89,46 +110,39 @@ which will result in the following import map.
 }
 ```
 
+Overrides may be useful for:
+- Polyfilling
+- Fixing a dependency with a local fork
+- Getting a dependency from a CDN instead
+
 ### Deletes
 
-You can apply deletions to the generated importMap by adding a `importMap` property to your package.json:
+You can apply deletions to the generated importmap by adding a `deletes` property to your package.json:
```

Comment:
I think it should be called `importMap` everywhere right?

Reply:
Actually should be "[import map](https://github.com/WICG/import-maps#the-import-map)" in running text. Should be `importMap` when referring to the property in package.json.

Fixed it